src/Task/FileSystem/CopyDir.php

```diff
@@ -32,7 +32,9 @@ protected function copyDir($src, $dst)
         if (false === $dir) {
             throw new TaskException($this, "Cannot open source directory '" . $src . "'");
         }
-        @mkdir($dst);
+        if (!is_dir($dst)) {
+            mkdir($dst, 0777, true);
```

Comment:
`permissions` option is not very descriptive, could you propose something better?
I agree to provide better option for that, and maybe 755 is a good idea for making it standard.